gax/test/showcase-echo-client/tsconfig.json

```diff
@@ -1,5 +1,5 @@
 {
-  "extends": "../../node_modules/gts/tsconfig-google.json",
+  "extends": "./node_modules/gts/tsconfig-google.json",
```

Comment:
double check the path, not sure if should change

Reply:
good call!